templates/Gemfile_custom.erb

```diff
@@ -34,7 +41,11 @@ group :development do
   gem 'capistrano-rails'
   gem 'capistrano-rbenv'
 <% end -%>
+<% if options[:bootstrap] -%>
```

Comment:
:+1:  já tam ještě rovnou generuju i mixin, protože jsem to nedávno potřeboval v media query a tam mi extend toho placeholderu byl k prdu https://github.com/blueberryapps/mrparkit.com/blob/master/app/assets/icons/_fontcustom.scss